app/src/main/cpp/ExternalVR.cpp

```diff
@@ -417,9 +417,12 @@ ExternalVR::PushFramePoses(const vrb::Matrix& aHeadTransform, const std::vector<
     for (int i = 0; i< controller.numButtons; ++i) {
       immersiveController.triggerValue[i] = controller.immersiveTriggerValues[i];
     }
+    for (int j = 0; j< controller.numButtons; ++j) {
+      immersiveController.triggerValue[j] = controller.immersiveTriggerValues[j];
+    }
     immersiveController.numAxes = controller.numAxes;
-    for (int i = 0; i< controller.numAxes; ++i) {
-      immersiveController.axisValue[i] = controller.immersiveAxes[i];
+    for (int j = 0; j< controller.numAxes; ++j) {
+      immersiveController.axisValue[j] = controller.immersiveAxes[j];
```

Comment:
I'm not sure what is going on here. It looks like trigger value is being set twice and in the last for loop `i` was replaced with `j`? I think this can be reverted? You could add a space between `i<` though.

Reply:
I just changed the name of the for variable as we were using the same name as in the outer for.